http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -394,7 +404,16 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         raw"config.batchAccountName=${landingZoneResources.batchAccountName.value}",
         raw"config.batchNodesSubnetId=${landingZoneResources.batchNodesSubnetName.value}",
         raw"config.drsUrl=${config.drsConfig.url}",
-        raw"config.workflowExecutionIdentity=${petManagedIdentity.id()}",
+        // raw"config.workflowExecutionIdentity=${petManagedIdentity.id()}",
+        /*
+        raw"config.landingZoneId=${}",
+        raw"config.subscriptionId=${}",
+
+        raw"config.batchAccountKey=${}",
+        raw"config.applicationInsightsAccountKey=${}",
```

Comment:
Just to note, this can be removed now and all parts of Helm chart can be updated to take `config.applicationInsightsConnectionString`.